src/hooks/use-toggle-state/use-toggle-state.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import { useCallback, useState } from 'react';
 
 const useToggleState = (defaultValue = true) => {
-  const [isToggled, setIsToggled] = React.useState(defaultValue);
-  const toggle = React.useCallback(() => {
-    setIsToggled(!isToggled);
-  }, [isToggled]);
+  const [isToggled, setIsToggled] = useState(defaultValue);
 
-  return [isToggled, toggle];
+  const setOn = useCallback(() => {
```

Comment:
Shouldn't we remove those extra functions?

Reply:
err yeah